packages/fields/src/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './fields';
 export * from './actions';
-export type * from './types';
+export { default as CreateTemplatePartModal } from './components/create-template-part-modal';
+export * from './types';
```

Comment:
Do we need to export the types?

Reply:
Currently, [we are using some types](https://github.com/WordPress/gutenberg/pull/67024/files#diff-22dafac83020324ae032e8217a80eabc59ddc3eedcb59c62d7353f4e49739404R18).

I'm going to export only the used one.

Reply:
Fixed with b9fb6f3a679dc9b05f4b7f5f38ce56b997852b8d

Reply:
It's a bit unfortunate that they show up in the docs (not in a great way)